recipes/fiji-simple_omero_client/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "fiji-simple_omero_client" %}
+{% set version = "5.12.2" %}
+ 
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GReD-Clermont/simple-omero-client/releases/download/{{ version }}/simple-omero-client-{{ version }}.jar
+  sha256: 246a006152ae51ff6eb5f0a62567083fe589c1f45413a4ed4e463d31604fcbea
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - fiji >=20220414
+    - fiji-omero_ij ==5.8.0
+
+test:
+  files:
+    - test.groovy
+
+about:
+  home: https://github.com/GReD-Clermont/simple-omero-client
+  license: GPLv2
+  license_file: LICENSE.txt
+  summary: Fiji/ImageJ plugin to simplify API to put/get objects on an OMERO server.
```

Comment:
It is actually not specific to ImageJ/Fiji (although this package is), and does not provide any plugin in fact. 
Would it make sense to rephrase it as such?
```yaml
summary: Java library to simplify API to put/get objects on an OMERO server from Fiji/ImageJ.
```